server_src/routes/settings.js

```diff
@@ -127,6 +132,11 @@ router.get("/server/update", ensureAuthenticated, async (req, res) => {
   let octoFarmUpdated = await SystemCommands.updateOctoFarm();
   res.send(octoFarmUpdated);
 });
+router.get("/server/update/check", ensureAuthenticated, async (req, res) => {
```

Comment:
Add API smoke-test. Look at supertest usage (test already present in test folder)